build.sbt

```diff
@@ -73,8 +74,8 @@ lazy val commonSettings = Seq(
     Developer("ruippeixotog", "Rui Gonçalves", "ruippeixotog@gmail.com", url("https://github.com/ruippeixotog")),
     Developer("derekmorr", "Derek Morr", "morr.derek@gmail.com", url("https://github.com/derekmorr"))),
 
+  crossScalaVersions := Seq(scala212, scala213, scala211),
   scalaVersion := crossScalaVersions.value.head,
```

Comment:
Nit: since the first version of `crossScalaVersions` is not guaranteed to be the "main" one anymore, can we define `scalaVersion := scala212` and sort `crossScalaVersions`? We can define a `mainScala` in `Dependency.Versions` if we really want to define it a single place, but I don't think it's necessary.

Reply:
Sure. Done that in cc447ee.